.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
Maybe something like "Create a report to help fix issues or provide feature requests"

Reply:
Updated!